Fix parsing libtool version on Debian.

--- a/third_party/fontconfig-2.17.1/autogen.sh
+++ b/third_party/fontconfig-2.17.1/autogen.sh
@@ -60,7 +60,7 @@
 
 have_libtool=false
 if $LIBTOOLIZE --version < /dev/null > /dev/null 2>&1 ; then
-	libtool_version=`$LIBTOOLIZE --version | sed 's/^.* \([0-9][.][0-9.]*\)[^ ]*$/\1/'`
+	libtool_version=`$LIBTOOLIZE --version | sed 's/^.* \([0-9][.][0-9.]*\)[^ ]*/\1/'`
 	case $libtool_version in
 	    1.4*|1.5*|1.6*|1.7*|2*)
 		have_libtool=true
